Core/HLE/sceUtility.cpp

```diff
@@ -332,8 +332,8 @@ static int UtilityWorkUs(int us) {
 	// This blocks, but other better priority threads can get time.
 	// Simulate this by allowing a reschedule.
 	if (us > 1000) {
-		hleEatMicro(us - 400);
-		return hleDelayResult(0, "utility work", 400);
+		hleEatMicro(1000);
+		return hleDelayResult(0, "utility work", us - 1000);
```

Comment:
For later: I see it's impossible here due to the surrounding if, but we should probably at least debug-assert that the delay amount is >= 0, in both hleDelayResult and hleEatMicro...